src/model/article-vor.v7.yaml

```diff
@@ -83,29 +83,6 @@ allOf:
                     format: uri
             required:
               - content
-        elifeAssessment:
```

Comment:
verified `article-vor.v6.yaml` and lower do not exist anymore and don't need to be maintained

Reply:
So should v7 be retired? https://prod--gateway.elifesciences.org/articles/100068 returns v8 in `Content-Type`

Reply:
We should discuss this sometime with @gnott present. This will not be addressed here.

Reply:
The reason I am touching v7 in this instance is because we are bringing this property into the snippet and I assumed it would create a class. I may reinstate this and test it out.